SpywareFilter/sections/general_url.txt

```diff
@@ -5,6 +5,11 @@
 ! Bad: .org/ads/$domain=example.org (for instance, should be in speficic.txt)
 !
 !
+! server-side GTM pattern B ex. https://load.ss.esade.edu/ykpgwaiz.js?st=5V4W3ZR
+!+ NOT_PLATFORM(ios, ext_safari)
+/\/\/([0-9a-z]+\.){1,}[0-9a-z]+\.[a-z]{2,}\/[a-z]{8}\.js\?st=[0-9A-Z]{6,8}$/$script,~third-party,match-case
+!+ PLATFORM(ios, ext_safari)
+/\/\/([0-9a-z]+\.)?[0-9a-z]+\.[0-9a-z]+\.[a-z][a-z]+\/[a-z]{8}\.js\?st=[0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z]+$/$~third-party,match-case
```

Comment:
`{8}` does not work in Safari. Suggested with `[a-z][a-z][a-z][a-z][a-z][a-z][a-z][a-z]`

```suggestion
/\/\/([0-9a-z]+\.)?[0-9a-z]+\.[0-9a-z]+\.[a-z][a-z]+\/[a-z][a-z][a-z][a-z][a-z][a-z][a-z][a-z]\.js\?st=[0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z]+$/$~third-party,match-case
```

Reply:
Great, thanks!